pkgs/development/ocaml-modules/pbkdf/default.nix

```
@@ -12,12 +12,13 @@ buildDunePackage rec {
 
   src = fetchFromGitHub {
     owner = "abeaumont";
-    repo = "ocaml-pbkdf";
+    repo = "ocaml-${pname}";
     rev = version;
     sha256 = "sha256-dGi4Vw+7VBpK/NpJ6zdpogm+E6G/oJovXCksJBSmqjI=";
   };
```

```suggestion
  src = fetchzip {
    url = "https://github.com/abeaumont/ocaml-pbkdf/releases/download/${version}/pbkdf-${version}.tbz";
    sha256 = "sha256-2WdCwqXEE9hMPk71j0tNa2j6jsn++Z8yU7g1GBOlPmA=";
  };
```
To fix version issue in META file.

---

On my understanding: `.tbz` files have metadata that is used in OCaml for version (check package's META file to understand). Ideally we should be fetching `.tbz` to retrieve the version metadata.
@vbgl Thoughts on @SuperSandro2000 feedback?